app/src/views/private/components/drawer-item.vue

```diff
@@ -333,7 +333,8 @@ function useRelation() {
 }
 
 function useActions() {
-	const { nestedValidationErrors } = useNestedValidation();
+	const { nestedValidationErrors, resetNestedValidationErrors } = useNestedValidation();
+	watch(internalActive, resetNestedValidationErrors);
```

Comment:
Nitpicking, but I guess we only need to call it once either on close / open:

```suggestion
	const { nestedValidationErrors, resetNestedValidationErrors } = useNestedValidation();

	watch(internalActive, (active) => {
		if (!active) resetNestedValidationErrors();
	});
```

Reply:
nitpicking but fair ❤️